ql/src/java/org/apache/hadoop/hive/ql/security/authorization/plugin/metastore/events/AlterTableEvent.java

```diff
@@ -101,6 +112,36 @@ private HiveOperationType getOperationType() {
       ret.add(getHivePrivilegeObjectDfsUri(newUri));
     }
 
+    if(newTable.getParameters().containsKey(hive_metastoreConstants.META_TABLE_STORAGE)) {
+      String storageUri = "";
+      DefaultStorageHandler defaultStorageHandler = null;
+      HiveStorageHandler hiveStorageHandler = null;
+      Configuration conf = new Configuration();
+      Map<String, String> tableProperties = new HashMap<>();
+      tableProperties.putAll(newTable.getSd().getSerdeInfo().getParameters());
```

Comment:
I am not sure I understand this logic. Why are the SerDe.getParameters() being added to Table.getParameters()? Is this what we always do?

Reply:
tableProperties is just a variable. I'm not setting this to the table parameters. The reason why I'm combining table parameters and serde parameters are, the user can give table properties like table name, column name or connection information in TBLPropperties or SERDE. So I need to put all the properties in a map and send the map to respective storage handler and extract table properties in it.